qrplatba/spayd.py

```diff
@@ -112,7 +112,7 @@ def _due_date(self):
         return ''
 
     def _format_item_string(self, item, name):
-        if item is not None:
+        if item:
             return "{name}:{value}*".format(name=name, value=item)
```

Comment:
Setting the message was skipped even when the message (`item`) was intentionally an empty string. Not always one wants to have a message filled in.